src/domain/card/CardExpirationDate/CardExpirationDate.stories.tsx

```
@@ -0,0 +1,86 @@
+    return (
+      <CardExpirationDate
+        cardExpirationDateRefs={cardExpirationDateRefs}
+        cardExpirationDate={cardExpirationDate}
+        cardExpirationDateErrorMessage={cardExpirationDateErrorMessage}
+        handleCardExpirationDateInputChange={handleCardExpirationDateInputChange}
+      />
+    );
+  },
+};
+
+export const Error = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)